package.json

```diff
@@ -1,7 +1,9 @@
 {
-  "name": "@zeta-apps-frontend/docs-site",
+  "name": "zetachain",
```

Comment:
```suggestion
  "name": "@zetachain/docs",
```

Reply:
I was just following [zetachain's pattern](https://github.com/zeta-chain/zetachain/blob/69cd5cfd1940610ed81a0e76e363d4ee3c2e46c3/package.json#L2) (a package.json that isn't published), but I suppose `@zetachain/docs` makes more sense even though we're not publishing it to npm.